### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#591] [Galaxy Trivia] [answered]
✨🟦🟨⬛
⬜🟦✝️
⬇️🟥⬜

**Comment by Veercodeprog** [comment]
The colors ✨🟦🟨⬛ correspond to the flag of the Republic of South Africa.
the second one is greece
The symbols ⬇️🟥⬜ correspond to the flag of Poland.

**Comment by peterbaccaro** [comment, marked as answer]
1) SAINT LUCIA
2) FINLAND
3) INDONESIA

![flags](https://user-images.githubusercontent.com/74666079/233293910-753844bf-ca45-4189-8838-03d6ecf65770.png)
